src/core/fieldformatter/qgsdatetimefieldformatter.cpp

```diff
@@ -21,13 +21,15 @@
 #include "qgsapplication.h"
 #include "qgsvariantutils.h"
 
-QString QgsDateTimeFieldFormatter::DATE_FORMAT = QStringLiteral( "yyyy-MM-dd" );
+const QString QgsDateTimeFieldFormatter::DATE_FORMAT = QStringLiteral( "yyyy-MM-dd" );
```

Comment:
Sorry I don't understand, can you explain the changes you would like to me to do ?

Reply:
I wanted to move the code to the header file by making the variables inline, but it's really not a big deal, this can be done another time and is not important.